### ISSUE TITLE
debconf: support 'seen' as a type

### ISSUE THREAD

**Issue by php-coder** [ansible/ansible-modules-extras#2456]
##### ISSUE TYPE

- Feature Idea
##### COMPONENT NAME

debconf
##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### OS / ENVIRONMENT

N/A
##### SUMMARY

I've found that to accept Oracle license I need to execute 2 debconf commands:

``` sh
echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections
echo debconf shared/accepted-oracle-license-v1-1 seen true | debconf-set-selections
```

The first has `select` type and it supported by `debconf` module while the second isn't supported.

To resolve this issue I suggest to add `seen` to the list of support types.
##### STEPS TO REPRODUCE

```
- name: Accept Oracle license
  debconf:
    name: "oracle-java8-installer"
    question: shared/accepted-oracle-license-v1-1
    vtype: seen
    value: true
```

##### ACTUAL RESULTS

```
TASK [php-coder.oraclejdk : Accept Oracle license] *****************************
fatal: [default]: FAILED! => {"changed": false, "failed": true, "msg": "value of vtype must be one of: string,password,boolean,select,multiselect,note,error,title,text, got: seen"}
```

**Comment by bcoca** [comment]
fixed via a28d7a00b899e88d9d0b349939ae85f18052bf5c , will be part of 2.2

**Comment by php-coder** [comment]
Thanks!
